lib/tasks/check_changelog.rake

```diff
@@ -0,0 +1,52 @@
+#--
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+#   This library is free software; you can redistribute it and/or modify
+# it only under the terms of version 2.1 of the GNU Lesser General Public
+# License as published by the Free Software Foundation.
+#
+#   This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+# details.
+#
+#   You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#++
+
+require 'rake'
+
+namespace "check" do
+  desc "Checking for new IDs (bugzilla,fate,...)  in *.changes file"
+  task :changelog do
+    begin
+      checkout
+      copy_sources
+
+      puts "Checking IDs in *.changes file" if verbose
+      # Checking makes only sense if the version in the *.spec file has been changed
+      if version_changed?( "#{osc_checkout_dir}/#{package_name}.spec" )
+        Dir.chdir(osc_checkout_dir) do
+          # Tags described in https://github.com/openSUSE/osc-plugin-factory/blob/e12bc02e9817277335ce6adaa8e8d334d03fcc5d/check_tags_in_requests.py#L63
+          cmd = "osc -A '#{obs_api}' diff *.changes | grep -i "\
+            "\"bnc#[0-9]\\+\\|"\
+            "fate#[0-9]\\+\\|"\
+            "boo#[0-9]\\+\\|"\
+            "bsc#[0-9]\\+\\|"\
+            "bgo#[0-9]\\+\\|"\
+            "cve-[0-9]\\{4\\}-[0-9]\\+\""
+          puts cmd if verbose
+          `bash -c '#{cmd}'`
```

Comment:
NP: What about doing that `grep` in pure Ruby instead of spawning an external process? Something like
```ruby
ret = `osc ...`
if ret.match(/(bnc|fate|boo|bsc|bgo)#[0-9]+/i) || ret.match(/cve-[0-9]{4}-[0-9]+/i)
```